recipes/autoesda/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "autoesda" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 83bfa06deaa41a29979918261fa69a1f53c623012b25d8e2cf1bd98fb3e597b8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - esda
+    - geopandas
+    - libpysal
+    - matplotlib
```

Comment:
If you're not using the Qt features of `matplotlib`, you could change this to `matplotlib-base`.

Reply:
Unfortunately the library makes use of the Qt features and Qt is listed as a dependancy of some of the autoesda dependancies